packages/core/src/scss/_partials/_colour-constants.scss

```diff
@@ -55,17 +79,19 @@ $link-hover: #005ea5;
 $link-active: $link-hover;
 
 // Alerts and Status - Dark
-$alert: $uq-red-dark20;
+$info: $uq-blue;
 $success: $uq-green-dark10;
+$warning: $uq-yellow;
```

Comment:
This also fails contrast requirements with white text.

Reply:
Based on the function rules set, any darker is too dull and muddy. Not sure how to solve this. Or we may have to break the rule as an odd case

Reply:
It's fine that the yellow variant requires dark text. But make sure that the developer knows that.